### PR TITLE
add _write_log()

### PR DESCRIPTION
**Pull request by jun-harashima**
This PR add `_write_log()` to group `_set_log_line()` and `logger.info(log_line)`.

### CODE REVIEW

File: msnc/model.py [hunk 1]
```diff
@@ -99,27 +99,22 @@ def run_training(
...
-                log_line = self._set_log_line(dev_accuracy, epoch, '[new best]')  # NOQA
+                self._write_log(dev_accuracy, epoch, '[new best]')
 
                 # Save epoch and performance information
                 self._best_epoch = epoch
                 self._best_dev_accuracy = dev_accuracy
 
                 if save_best_model:
                     self._save('best.model')
-                    logger.debug('Update best model')
```
Comment (jun-harashima) [inline msnc/model.py]: Is this necessary? 🤔

File: msnc/model.py [hunk 2]
```diff
@@ -99,27 +99,22 @@ def run_training(
...
 
-                logging.info(log_line)
```
Comment (jun-harashima) [inline msnc/model.py]: Are there any differences between `logger.info(log_line)` and `logging.info(log_line)`? 🤔

**Comment by jun-harashima** [comment]
@himkt please check the changes 🙇

File: msnc/model.py (hunk 1 above)
Comment (himkt) [reply msnc/model.py]: Good question. Actually this message is not needed or should be put on '__init__'.

File: msnc/model.py (hunk 2 above)
Comment (himkt) [reply msnc/model.py]: logging.info shuld be replaced with logger.info
I'm sorry. 🙇 :bow

**Pull request merged**